extensions/ql-vscode/src/view/model-editor/LibraryRow.tsx

```diff
@@ -14,6 +14,7 @@ import {
 } from "@vscode/webview-ui-toolkit/react";
 import type { ModelEditorViewState } from "../../model-editor/shared/view-state";
 import type { AccessPathSuggestionOptions } from "../../model-editor/suggestions";
+import { getCandidates } from "../../model-editor/auto-model";
```

Comment:
I'm getting lots of errors around types when compiling the view and I wonder if this import is causing it. The `auto-model.ts` file does import a bunch of other stuff too, even though `getCandidates` itself it pretty innocent.
```
../codeql-cli/cli.ts:10:8 - error TS1259: Module '"/Users/robertbrignull/github/vscode-codeql/extensions/ql-vscode/node_modules/tree-kill/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag

10 import tk from "tree-kill";
          ~~

  ../../node_modules/tree-kill/index.d.ts:13:1
    13 export = treeKill;
       ~~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.

../databases/local-databases/database-manager.ts:2:8 - error TS1192: Module '"vscode"' has no default export.

2 import vscode from "vscode";
         ~~~~~~

../databases/local-databases/database-manager.ts:73:31 - error TS7006: Parameter 'e' implicitly has an 'any' type.

73     const disposable = event((e) => {
                                 ~

../databases/local-databases/database-manager.ts:507:65 - error TS7006: Parameter 'folder' implicitly has an 'any' type.

507     return (vscode.workspace.workspaceFolders || []).findIndex((folder) =>
                                                                    ~~~~~~

../databases/local-databases/database-manager.ts:576:8 - error TS7006: Parameter 'folder' implicitly has an 'any' type.

576       (folder) => item.belongsToSourceArchiveExplorerUri(folder.uri),
           ~~~~~~

../local-queries/results-view.ts:198:55 - error TS7031: Binding element 'kind' implicitly has an 'any' type.

198       this.databaseManager.onDidChangeDatabaseItem(({ kind }) => {
                                                          ~~~~

../packaging/qlpack-file-loader.ts:6:34 - error TS2732: Cannot find module './qlpack-file.schema.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.

6 import qlpackFileSchemaJson from "./qlpack-file.schema.json";
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 7 errors in 4 files.

Errors  Files
     1  ../codeql-cli/cli.ts:10
     4  ../databases/local-databases/database-manager.ts:2
     1  ../local-queries/results-view.ts:198
     1  ../packaging/qlpack-file-loader.ts:6
```